adapter.js

```diff
@@ -1,207 +1,82 @@
-
-'use strict';
-
+const PostgresAdapter = require('realm-data-adapters').PostgresAdapter
+const Realm = require('realm')
 const fs = require('fs');
 const path = require('path');
-const Realm = require('realm');
-const PostgresAdapter = require('./postgres_adapter.js');
-const PostgresClient = require('libpq');
-
-// CONFIGURABLE
-var access_token = 'ewoJImlkZW50aXR5IjogIl9fYXV0aCIsCgkiYWNjZXNzIjogWyJ1cGxvYWQiLCAiZG93bmxvYWQiLCAibWFuYWdlIl0KfQo=:UH4womynqiP+9/b4kXvGc7E9XrPB3bbdCKJNXhjbWwXN7jbRLlldsG5jCu5CF9vEIYl8nSrORmGpIC/6tztlPzKs3RkIphJWb19uvD3shL6zKy7W+iZrGzPrFDNpEXM4hp4Hq1L/zgxMf1vlElL5JzW2Yq5AybpZZAeP6IVM/xC5bRfCzf7iER8om28WxnVK3CC2buOJbV+dKKWNSVbVl7Ol3N9+Vu2owkByGGGaY0DjzWkTzf+Y+q8r1aoC5MBgOqXhde+bWSiKc2hzFK5FZH1m0KG5+odD40r2mahxdAw304S6ipJcIwGA5pFlBsSNQ/c9K8DYvR0sUZXu9M5xlQ==';
-
-var pg_host = 'localhost';
-var pg_port = 5432;
-var pg_user = 'afish';
-var pg_password = 'password';
-
-// DO NOT CHANGE UNLESS INSTRUCTED
-var pg_dbName = 'dvdrental';
-
-// Create a Postgres client to setup DVD database
-var client = new PostgresClient();
-var connectionString = 'dbname='+pg_user+' host='+pg_host+' port='+pg_port+' user='+pg_user+' password='+pg_password;
-client.connectSync(connectionString);
-
-// HACK - delete admin realm
-try {
-  require( 'child_process' ).execSync('rm -r realm-object-server/adapter/__admin.realm*');
-}
-catch(e) {}
-
-var args = process.argv;
-args.splice(0,2);
-if (args.length > 0) {
-  for (let index in args) {
-    let val = args[index];
-    if (val == '-reset' ||
-        val == '-r') {
-          console.log(`Dropping replication slot '${pg_dbName}'`);
-          client.exec(`SELECT pg_drop_replication_slot('${pg_dbName}');`);
-          console.log(`Removing ${pg_dbName} database`);
-          client.exec(`DROP DATABASE IF EXISTS ${pg_dbName}`);
-          
-          var deleteFolderRecursive = function(path) {
-            if( fs.existsSync(path) ) {
-              console.log(`Removing ${path}`);
-              fs.readdirSync(path).forEach(function(file,index){
-                var curPath = path + "/" + file;
-                if(fs.lstatSync(curPath).isDirectory()) { // recurse
-                  deleteFolderRecursive(curPath);
-                } else { // delete file
-                  fs.unlinkSync(curPath);
-                }
-              });
-              fs.rmdirSync(path);
-            }
-          };
-          
-          deleteFolderRecursive('./realm-object-server');
-          process.exit()
-    }
-  } 
-}   
-
-process.on('uncaughtException', function(err) {
-  console.log(err);
-});
-
-// Check for dvdrental 
-client.exec(`SELECT EXISTS (SELECT 1 from pg_database WHERE datname='${pg_dbName}')`);
-var existValue = client.getvalue(0, 0);
-if (existValue == 'f') {
-  console.log('Sample Database Not Found!');
-  console.log('--> Creating %s database', pg_dbName);
-  client.exec(`CREATE DATABASE ${pg_dbName};`);
-  
-  // Load the sample data
-  console.log('--> Loading sample data');
```

Comment:
Why do we still need this? We have this now:
`createPostgresDB: false,` I can set this to `true` if we need to?